app/models/story_search.rb

```diff
@@ -38,9 +51,13 @@ def parse(query_params)
     end
   end
 
+  def valid?(conditions)
+    conditions.keys.map { |value| StorySearch.operand.find_value(value) }.exclude? nil
```

Comment:
Can this be replaced with this?

```suggestion
    conditions.keys.all? { |value| StorySearch.operand.find_value(value) }
```